package.json

```diff
@@ -3,19 +3,24 @@
     "version": "1.0.0",
     "description": "Testing the LAPIS API, backed by SILO, designed for easily adding new test-cases.",
     "scripts": {
-        "test": "vitest --reporter=html",
+        "test": "vitest --reporter=html --reporter=default",
         "format": "prettier --write \"./**/*.{ts,tsx,json,astro,md,mdx,mjs,cjs}\"",
-        "check-format": "prettier --check \"**/*.{ts,tsx,json,astro,md,mdx,mjs,cjs}\""
+        "check-format": "prettier --check \"**/*.{ts,tsx,json,astro,md,mdx,mjs,cjs}\"",
+        "check-types": "tsc --noEmit",
+        "check": "npm run check-format && npm run check-types"
     },
     "dependencies": {
-        "@mongodb-js/zstd": "^1.2.0",
-        "@types/node": "^22.5.3",
-        "prettier": "^3.3.3",
-        "typescript": "^5.5.4",
-        "vitest": "^2.0.5"
+        "@mongodb-js/zstd": "^1.2.1"
     },
     "type": "module",
     "devDependencies": {
+        "@types/node": "^22.5.4",
+        "prettier": "^3.3.3",
+        "typescript": "^5.5.4",
+        "vitest": "^2.0.5",
```

Comment:
IMO we don't need to distinguish in dev and prod dependencies - either everything should be dev or everything should be prod, since this is a testing package. You could argue:
* The purpose of this package is testing. So the testing packages should be prod dependencies.
* There is no real "production usage", so everything should be a devDependency.